components/Carousel.js

```diff
@@ -1,11 +1,16 @@
 // import Brews from "./Brews";
 // import Blog from "./Blog";
 import About from "./About";
-// import TapList from "./TapList";
+import TapList from "./TapList";
 // import Events from "./Events";
 import Nav from "../components/Nav";
 import Section from "./Section";
 
+const bigStyle = {
+  width: "200vw",
+  backgroundColor: "#ededed"
```

Comment:
You'll also need to override the `flex-basis` (defaults to `100%` in the `Section` component) with `auto` I think?